t/t8100-git-survey.sh

```diff
@@ -34,6 +34,26 @@ test_expect_success 'git survey (default)' '
 	git survey --all-refs >out 2>err &&
 	test_line_count = 0 err &&
 
+	test_oid_cache <<-EOF &&
```

Comment:
Thanks for this. I bet it was a hassle to create.

Reply:
Hehe, it was! But I didn't do it today, I did it in the context of #5146 :-)